authentication/managing_cloud_provider_credentials/cco-mode-manual.adoc

```diff
@@ -26,11 +26,11 @@ An AWS, global Azure, or GCP cluster that uses manual mode might be configured t
 [id="additional-resources_cco-mode-manual"]
 == Additional resources
 
-* xref:../../installing/installing_alibaba/manually-creating-alibaba-ram.adoc#manually-creating-alibaba-ram[Manually creating RAM resources for Alibaba Cloud]
```

Comment:
@jianli-wei I know you have ack'ed this one. but can you re-confirm this one?

Reply:
@snarayan-redhat Do you mean we need to update the figure and remove Alibaba Cloud from the text?

Reply:
@jianli-wei yes, so basically the question would be whether manual mode and CCO are still applicable to Alibaba when installed using AI.

Reply:
During the testing of installing OpenShift on Alibaba Cloud using Assisted Installer, I did not create credentials manually, so I think it's not required CCO in manual mode. Give me some time to double confirm, will update you then. Thanks!

Reply:
Yeah, that will help, because the explanation of the diagram says:
![image](https://github.com/openshift/openshift-docs/assets/84562993/4f534251-f5ce-49bb-871b-b2cfc7e85420)
Since there were no changes to Updating book confirmed, this wasn't changed. But how I related it to the change is, if we are not going to use `ccoctl` then this might have to be changed.

Reply:
FYI I just tried using Assisted Installer to install OpenShift 4.16.0-rc.4 on Alibaba Cloud, it looks like unable to determine the CCO mode. On considering I did not create credentials manually, we could `update the figure and remove Alibaba Cloud from the text`. Thanks! 

```
$ oc get clusterversion
NAME      VERSION       AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.16.0-rc.4   True        False         7m21s   Cluster version is 4.16.0-rc.4
$ oc get nodes
NAME                         STATUS   ROLES                         AGE   VERSION
jiwei-0614z-ali-instance-1   Ready    control-plane,master,worker   21m   v1.29.5+87992f4
$ oc get co
NAME                                       VERSION       AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
authentication                             4.16.0-rc.4   True        False         False      7m56s   
baremetal                                  4.16.0-rc.4   True        False         False      18m     
cloud-controller-manager                   4.16.0-rc.4   True        False         False      12m     
cloud-credential                           4.16.0-rc.4   True        False         False      17m     
cluster-autoscaler                         4.16.0-rc.4   True        False         False      17m     
config-operator                            4.16.0-rc.4   True        False         False      19m     
console                                    4.16.0-rc.4   True        False         False      8m12s   
control-plane-machine-set                  4.16.0-rc.4   True        False         False      18m     
csi-snapshot-controller                    4.16.0-rc.4   True        False         False      19m     
dns                                        4.16.0-rc.4   True        False         False      18m     
etcd                                       4.16.0-rc.4   True        False         False      14m     
image-registry                             4.16.0-rc.4   True        False         False      11m     
ingress                                    4.16.0-rc.4   True        False         False      18m     
insights                                   4.16.0-rc.4   True        False         False      18m     
kube-apiserver                             4.16.0-rc.4   True        False         False      11m     
kube-controller-manager                    4.16.0-rc.4   True        False         False      11m     
kube-scheduler                             4.16.0-rc.4   True        False         False      12m     
kube-storage-version-migrator              4.16.0-rc.4   True        False         False      19m     
machine-api                                4.16.0-rc.4   True        False         False      14m     
machine-approver                           4.16.0-rc.4   True        False         False      18m     
machine-config                             4.16.0-rc.4   True        False         False      19m     
marketplace                                4.16.0-rc.4   True        False         False      18m     
monitoring                                 4.16.0-rc.4   True        False         False      8m49s   
network                                    4.16.0-rc.4   True        False         False      19m     
node-tuning                                4.16.0-rc.4   True        False         False      19m     
openshift-apiserver                        4.16.0-rc.4   True        False         False      12m     
openshift-controller-manager               4.16.0-rc.4   True        False         False      11m     
openshift-samples                          4.16.0-rc.4   True        False         False      11m     
operator-lifecycle-manager                 4.16.0-rc.4   True        False         False      18m     
operator-lifecycle-manager-catalog         4.16.0-rc.4   True        False         False      18m     
operator-lifecycle-manager-packageserver   4.16.0-rc.4   True        False         False      11m     
service-ca                                 4.16.0-rc.4   True        False         False      19m     
storage                                    4.16.0-rc.4   True        False         False      19m     
$ oc logs -n openshift-cloud-credential-operator cloud-credential-operator-79bc865b6b-m2dhk -c cloud-credential-operator | grep mode
time="2024-06-14T06:31:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:33:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:35:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:37:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:39:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:41:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:43:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:45:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:47:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:49:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:51:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
time="2024-06-14T06:53:15Z" level=info msg="unsupported cloud for determining CCO mode" cloud=None controller=metrics
$

Reply:
Thanks for the confirmation @jianli-wei !